.storybook/main.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  'staticDirs': ['../www/public/static/'],
+  'stories': [
+    '../src/**/*.stories.mdx',
+    '../src/**/*.stories.@(js|jsx|ts|tsx)'
+  ],
+  'addons': [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-interactions',
+    'storybook-stylesheet-toggle'
+  ],
+  'framework': '@storybook/react',
+  'core': {
+    'builder': '@storybook/builder-webpack5'
+  }
+}
```

Comment:
Please add an empty line to the end of the page

Reply:
added, thanks